### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by stevessr** [iRedScarf/onedrive-index#16] [Q&A] [answered]
启用totp后报错
data: {
      error: 'invalid_grant',
      error_description: "AADSTS50076: Due to a configuration change made by your administrator, or because you moved to a new location, you must use multi-factor authentication to access '00000003-0000-0000-c000-000000000000'. Trace ID: bc652aa9-e531-4f6b-a955-64c5d1fc2800 Correlation ID: 3dd21038-db55-4407-9b74-c4f13b4eaf02 Timestamp: 2024-01-21 05:23:33Z",
      error_codes: [Array],
      timestamp: '2024-01-21 05:23:33Z',
      trace_id: 'bc652aa9-e531-4f6b-a955-64c5d1fc2800',
      correlation_id: '3dd21038-db55-4407-9b74-c4f13b4eaf02',
      error_uri: 'https://login.microsoftonline.com/error?code=50076',
      suberror: 'basic_action'
    }
  }
}

**Comment by iRedScarf** [comment]
请再具体描述下错误发生时的情景，部署进行到哪一步时发生的错误？

**Reply by stevessr** [reply, marked as answer]
sorry，已经解决，是更改安全设置后导致的refresh token 失效所致
